public/js/controllers/app_controller.js

```diff
@@ -91,6 +94,21 @@ class Controller {
         });
     }
 
+    static displayPipelines() {
```

Comment:
qq - how do we restrict this to active project pipelines only?

Reply:
@IsaackMwenda, cc @as2388 I suggest we add a "project_name" field in the documents then I can update the query performed on firestore to filter out pipelines for inactive projects.